core/services/chainlink/application.go

```diff
@@ -404,8 +404,8 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 	// until app.LogBroadcaster.DependentReady() call (see below)
 	logBroadcaster.AddDependents(1)
 
-	for _, service := range app.subservices {
-		if err = app.HealthChecker.Register(reflect.TypeOf(service).String(), service); err != nil {
+	for _, r := range app.svcs.Runners {
+		if err = app.HealthChecker.Register(reflect.TypeOf(r.svc).String(), r.svc); err != nil {
```

Comment:
This could also be included in the abstraction since all services are guaranteed to implement the checker

Reply:
Will look into moving it once I get these tests passing again